smslibrary/src/test/java/com/eis/smslibrary/SMSSentBroadcastReceiverTest.java

```diff
@@ -0,0 +1,97 @@
+package com.eis.smslibrary;
+
+import android.app.Activity;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.telephony.SmsManager;
+
+import com.eis.smslibrary.listeners.SMSSentListener;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+public class SMSSentBroadcastReceiverTest {
+
+    private SMSSentBroadcastReceiver receiver;
+    private SMSSentListener listenerMock = mock(SMSSentListener.class);
+    private Context contextMock = mock(Context.class);
+    private final SMSPeer VALID_PEER = new SMSPeer("+393423541601");
+    private final String VALID_TEXT = "Some random text to send";
+    private final SMSMessage VALID_MESSAGE = new SMSMessage(VALID_PEER, VALID_TEXT);
+
+    @Before
+    public void setup(){
+        receiver = spy(new SMSSentBroadcastReceiver(VALID_MESSAGE, listenerMock));
+    }
+
+    @Test
+    public void constructor_doesntThrow(){
+        new SMSSentBroadcastReceiver(VALID_MESSAGE, listenerMock);
```

Comment:
The "automatic code reviewer" is in right.
Here should write:
`try {`
     `new SMSSentBroadcastReceiver(VALID_MESSAGE, listenerMock);`
     `//Success }`
`catch (Exception e) `{
     `Assert.fail(...); }`